### DISCUSSION TITLE
https://notes.junghanacs.com/

### DISCUSSION THREAD

**Discussion by giscus** [junghanacs/notes.junghanacs.com#3] [Announcements]
# https://notes.junghanacs.com/

Meditations on Technology, Learning, Life, and Text-editor 브레인덤프 : 디지털가든 브레인덤프가 더 어울릴 것 입니다. 아. 내보내려니 이게 자기 검열을 하게 되는 군요. 대분류 - 디렉토리 : 태그 디렉토리 인덱스 notes 태그 인덱스 journal 조직모드(Org-Mode) 등등 추가 주제별 : 십진분류 KDC 한국십진분류 이맥스 : Obsidian Logseq 공사중 AI 지식 베이스 : NotebookLM 공사중 홈페이지 : 브레인덤프 Authorlogy Life - junghanacs.com 공사중 노트 통계 899 notes : 2024-09-07 RSS 주소 RSS:Junghanacs's BrainDump Last revised and exported on 2024-09-08 21:29:25 +0900 브레인덤프 디지털가든 레퍼런스 Index - braindump.jethro.dev .

https://notes.junghanacs.com/

**Comment by sprachlust** [comment]
Love your digital garden! Looking forward to new posts from the Text Master :)

**Comment by junghanacs** [comment]
랜덤 노트 기능 부탁합니다. 화장실에서 활용하기 좋을 것 같네요.

**Reply by junghan0611** [reply]
최근 넣었습니다 애용바랍니다. 힣

**Comment by junghan0611** [comment]
AIONS 클럽 인터내쇼날의 B의 추천으로 알베르카뮈의 이방인을 다 들었다. 오늘 엄마가 돌아가셨다만 몇번 듣다가 말았는데 이번엔 다 들었다. 햇살이 참 좋아서 빵빵했다는 말에 으악. 주인공은 에이죤트인가. 동서남북을 한칸씩 오간 뒤에 제자리에서 둥근원을 그리고는 하나를 말하는가? 아. 기억나지 않는다. 아니 기억하고 싶지 않다. 덩어리로 온 카뮈표 이야기는 스미며 정합성 테스트 중이다. 딱딱. 아주 일치합니다. 이런. 주머니에 권총이 있다면? 다행이다. 없어서 다행이로다.

요즘 업댓이 없어서 아쉬운 클럽 분들을 위해 남깁니다. 장황한 업데이트가 있을 예정입니다. 힣이 생존이 어려운 상황입니다. 클럽 여러분의 압도적인 서포트 부탁합니다.

빠라삐리뽀 프로젝트 이며 부제 힣을 지켜라! 입니다.

**Comment by junghan0611** [comment]
https://www.threads.com/@junghanacs
힣은 요즘 근황은 수레드에 있어요. 인간지능 인공지능의 대결합을 위한 엑스칼리버를 담금질하느라 바쁘군요. 스레드 구독은 하지마세요. 100명 넘어갈까봐 걱정 입니다. 아. AIONS 클럽 인터내쇼날 형제자매님들 연락 좀 하세요. 그럼 20000
